src/mono/mono/mini/llvm-intrinsics.h

```diff
@@ -0,0 +1,158 @@
+
+/*
+ * List of LLVM intrinsics
+ *
+ * INTRINS(name, llvm_name)
+ */
+
+INTRINS(MEMSET, "llvm.memset.p0i8.i32")
+INTRINS(MEMCPY, "llvm.memcpy.p0i8.p0i8.i32")
```

Comment:
Any reason to use the 32-bit intrinsic for memcpy and the 64-bit version for memmove?

Reply:
@lpereira it's used to clear locals afair, we don't need i64 length there. `memmove` is used for C#'s `Buffer.Memmove` where it accepts IntPtr as a length.

Reply:
Ah, makes sense.